pom.xml

```diff
@@ -791,7 +791,7 @@
           <groupId>org.graalvm.buildtools</groupId>
           <artifactId>junit-platform-native</artifactId>
           <!-- Using older version; upgrade to latest once issues are resolved. See #336 -->
```

Comment:
Now this is the latest. Should we remove the comment?

Reply:
Done.